src/server/cluster/cluster_family.cc

```diff
@@ -811,7 +808,7 @@ void ClusterFamily::InitMigration(CmdArgList args, ConnectionContext* cntx) {
 
   auto [source_id, flows_num] = parser.Next<std::string, uint32_t>();
 
-  SlotRanges slots;
+  std::vector<SlotRange> slots;
```

Comment:
It'd be cool to use `SlotRanges` here as well. You'll probably need to have an `Extend()` (preferably renamed) version that accepts a single `SlotRange` though..

Reply:
in this case, I need to sort it every time

Reply:
well, if you end up with an implementation based on `SlotSet` you could change this, but anyway it's not very important

Reply:
I think we can rewrite this code with SlotSet and provide SlotSet into the Outgoing and Incoming migrations instead of SlotRanges. But it's additional refactoring that isn't needed now